extension/src/popup/components/signTransaction/Operations/index.tsx

```diff
@@ -255,8 +255,7 @@ export const Operations = ({
   const { t } = useTranslation();
 
   enum AuthorizationMap {
-    "",
-    "Authorization Required",
+    "Authorization Required" = 1,
```

Comment:
oof, this was a very weird error that Jest did not like; it didn't like having an empty string at key 0. This is actually fine bc Horizon never returns 0 for flags